its/projects/abacus/pom.xml

```diff
@@ -54,6 +54,7 @@
     <sonar.pluginName>Abacus</sonar.pluginName>
     <sonar.pluginClass>org.sonar.plugins.abacus.AbacusPlugin</sonar.pluginClass>
     <sonar.scm.disabled>true</sonar.scm.disabled>
+    <argLine> --add-opens java.base/java.lang=ALL-UNNAMED</argLine>
```

Comment:
Shall we remove the extra space at the beginning?

Reply:
Yes totally!